soweego/wikidata/sparql_queries.py

```diff
@@ -414,3 +414,43 @@ def query_birth_death(qids_bucket):
     query += """}"""
 
     return query
+
+
+def get_subclasses_of_qid(QID: str) -> Set[str]:
+    """
+    Given a QID, return the QID of it's subclasses
+
+    :param qid: QID we want to get the subclasses for
+
+    :return: set with QIDs of subclasses
+    """
+
+    # subclasses (?res is subclass of QID)
+    result = make_request("""
+                SELECT DISTINCT ?res WHERE {
+                    ?res wdt:P279* wd:%s .
+                } 
+            """ % QID)
+
+    return set(item['?res'].strip('>').split('/')[-1]
```

Comment:
Have you considered reusing the private function `_get_valid_qid`?

Reply:
I wasn't aware of that function :P thanks! Done in 1a3d871